implementations/rust/ockam/ockam_core/src/routing/route.rs

```diff
@@ -19,6 +19,15 @@ impl Route {
         RouteBuilder::new()
     }
 
+    /// Create a route from a Vec of addresses
+    pub fn create<T: Into<Address>>(vt: Vec<T>) -> Self {
```

Comment:
Can we replace vector with slice?

Reply:
Hmmm, I'll experiment. That would be cool.

Reply:
Couldn't figure out a way. I also couldn't get it working even with a static constructor, in a meaningful way. I can declare `fn create<T: Into<Address>>(slice: &[T])` but when one iterates over the slice it yields `&T` instead of `T` itself, so can't call '`into()`. Maybe could figure out something with more playing.